### PR TITLE
PostGreSql: added java readme

### PR DESCRIPTION
**Pull request by iscai-msft**

### CODE REVIEW

File: specification/postgresql/resource-manager/readme.java.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+``` yaml $(java) && $(multiapi)
+batch:
+  - tag: package-2017-12-01-preview
+  - tag: package-2017-12-01
+```
+
+### Tag: package-2017-12-01-preview and java
+
+These settings apply only when `--tag=package-2016-12-01-preview --java` is specified on the command line.
```
Comment (erich-wang) [inline specification/postgresql/resource-manager/readme.java.md]: Is it typo for 2016?

File: specification/postgresql/resource-manager/readme.java.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+java:
+  namespace: com.microsoft.azure.management.postgresql.v2017_12_01_preview
+  output-folder: $(azure-libraries-for-java-folder)/postgresql/resource-manager/v2017_12_01_preview
+regenerate-manager: true
+generate-interface: true
+```
+
+### Tag: package-2017-12-01 and java
+
+These settings apply only when `--tag=package-2016-12-01 --java` is specified on the command line.
```
Comment (erich-wang) [inline specification/postgresql/resource-manager/readme.java.md]: Is it typo for 2016?

**Review by erich-wang — CHANGES_REQUESTED** [review]

File: specification/postgresql/resource-manager/readme.java.md (hunk 1 above)
Comment (iscai-msft) [reply specification/postgresql/resource-manager/readme.java.md]: yes, thank you so much for catching this!

File: specification/postgresql/resource-manager/readme.java.md (hunk 2 above)
Comment (iscai-msft) [reply specification/postgresql/resource-manager/readme.java.md]: Fixed this one too, thanks again for catching!

**Review by erich-wang — APPROVED** [review]

**Pull request merged**